connector/oidc/oidc.go

```diff
@@ -209,6 +216,10 @@ func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string)
 		opts = append(opts, oauth2.SetAuthURLParam("hd", preferredDomain))
 	}
 
+	if len(c.acrValues) > 1 {
```

Comment:
```suggestion
	if len(c.acrValues) > 0 {
```

Reply:
Some providers use acr values like `0` or `1`.
See https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens

Reply:
Done, the check if the string is empty should really be 0! my bad 😅